bridges/snowbridge/primitives/router/src/outbound/mod.rs

```diff
@@ -407,3 +413,42 @@ where
 		Ok((Command::MintForeignToken { token_id, recipient, amount }, *topic_id))
 	}
 }
+
+/// An adapter for the implementation of `ExporterFor`, which attempts to find the
+/// `(bridge_location, payment)` for the requested `network` and `remote_location` and `xcm`
+/// in the provided `T` table containing various exporters.
+pub struct NetworkWithXcmExportTable<T, M>(core::marker::PhantomData<(T, M)>);
```

Comment:
I like the way you implemented this predicate. Maybe a better name would be:
```suggestion
pub struct XcmFilterExporter<T, M>(core::marker::PhantomData<(T, M)>);
```

Reply:
https://github.com/paritytech/polkadot-sdk/pull/6074/commits/47d26e8c1b25126761956995bbc1692965944438